pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+## Instructions and Submission Templates
+Three types of predictions will be accepted. Participants are encouraged to submit their results in all or multiple submission types as it fits to their prediction methods.
+
+#### Type I - microscopic pKas and related microstates
+Predicting microscopic pKa values and related microstates structures. Different protonation states and tautomers constitute different microstates. 
+- Fill the `typeI_microscopic_pKas_and_microstates.csv` template for all molecules.
+- For each molecule, report as many microscopic pKas based on your best guest.
+- Record the pair of microstates IDs of microstate structures pairs (protonated HA and deprotonated A) associated for each microscopic pKa. To determine the microstate ID for your predicted structure check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates).
+- If your predicted structure is not included in the list, contact us to [make a request for new microstate](mehtap.isik@choderalab.org). See more details are in missing microstates section.
+- Report microscopic pKa values with 2 decimals (e.g. 10.71).
+- Reporting standard error of the mean (SEM) is optional and should be reported with 2 decimals. (e.g. 1.02).
+
+#### Type II - microstate populations as a function of pH
+Predicting fractional population of microstates between pH interval 2 to 12 with 0.1 pH increments, based on microscopic pKa predictions.
+- Fill the `typeII_microstate_fractional_populations.csv` template file for all molecules and microstates you have predictions for.
+- To determine the microstate ID for your predicted microstate populations check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates).
+- If your predicted structure is not included in the list, contact us to [make a request for new microstate](mehtap.isik@choderalab.org). See more details are in missing microstates section.
+- Report natural logaritm of fractional microstate populations with scientific notation and 3 decimals of precision, for each pH.  
+e.g. For a molecule with only two possible microstates A and B `ln(fractional microstate population) = ln(N_A/(N_A+N_B))` where `N_A` and `N_B` represent number of molecules at microstate A and B.   
+At a pH where 90.0% of the molecules are in microstate B and 10.0% of molecules are in state A  `ln(fractional microstate A population) = ln(0.100/(0.100+0.900)) = -2.30E+00`.  
+- If your estimate of `fractional microstate population` is 0, thus `ln(fractional microstate population) = ln(0)`, report as `-infinity`.
+- Do not report SEM in this submission type.
+- For pH value or microstates which you don't have any estimates for, leave the csv table cell empty. 
```

Comment:
For pH values or microstates which you don't have an estimate, leave that line or cell of the csv table empty.

Reply:
Is this true for all three formats? You should include this point in each section if it is since people may only look at the format they plan to submit to.

Reply:
Very good point. I will do so.

Reply:
Done.